### ISSUE TITLE
missing timestamps for replaying

### ISSUE THREAD

**Issue by DiTo97** [Improbable-AI/VisionProTeleop#9]
it would be nice to replay a recorded trajectory from the AVP device.

ATM it's impossible as the recording only stores the sequence of transformations without the timestamps.

**Comment by Yaxit** [comment]
Yes, would be nice to have the on-device timestamps, similarly to how it it in the recording format.
